csrankings-h.csv

```diff
@@ -733,6 +733,7 @@ Hongjie Chen,Peking University,http://www.sei.pku.edu.cn/people/chenhj,NOSCHOLAR
 Hongjin Liang 0001,Nanjing University,https://cs.nju.edu.cn/hongjin,c-jKd-UAAAAJ
 Hongkai Wen 0001,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/Hongkai_Wen,SSbjWKAAAAAJ
 Hongkai Yu,Cleveland State University,http://cis.csuohio.edu/~h.yu/me.html,JnQts0kAAAAJ
+Honglak Lee,University of Michigan,https://web.eecs.umich.edu/~honglak/,fmSHtE8AAAAJ

```

Comment:
Senior VP and Chief Scientist at LG AI (https://www.linkedin.com/in/honglak-lee-8b07709/). This sounds incompatible with a full-time (>= 75%) appointment at Michigan. Please clarify.

Reply:
I know that he is back from leave. Anecdotally I see him in the computer science building roughly every week or so and know he is actively advising students (at least 3-5 right now), taking on new students, and teaching courses 1 semester a year.

From his teaching tab:
"Teaching
EECS 545: Machine Learning, Winter 2022
EECS 545: Machine Learning, Winter 2021
EECS 545: Machine Learning, Winter 2020
EECS 598: Deep Learning, Winter 2019"

Reply:
I'll need more than that.

Reply:
Makes sense! Thanks for all the hard work ☺️.